# Empty message, gets published when the timer reaches zero. All nodes that need to stop
# after when the game finishes will listen to this notification.